torch/_torch_docs.py

```diff
@@ -1130,6 +1130,31 @@ def merge_dicts(*dicts):
     tensor([ True, False, False])
 """.format(**common_args))
 
+add_docstr(torch.broadcast_to,
+           r"""
+broadcast_to(input, size) -> Tensor
+
+Broadcasts the input tensor to a new shape according to :ref:`broadcasting-semantics`.
```

Comment:
"Broadcasts :attr:\`input\` to the shape :shape:\`shape`. Equivalent to calling \`input.expand(shape)\`. See :meth:\`~Tensor.expand\` for details.

Reply:
Thanks so much for this suggestion, the function description has been updated.
[rendered doc for torch.broadcast_to](https://user-images.githubusercontent.com/40605299/101917736-86db0080-3c03-11eb-9409-016d9b9c03c6.png)